psycopg/connection_type.c

```diff
@@ -1395,7 +1395,15 @@ connection_setup(connectionObject *self, const char *dsn, long int async)
         PyObject *ptype = NULL, *pvalue = NULL, *ptb = NULL;
         PyErr_Fetch(&ptype, &pvalue, &ptb);
         obscure_password(self);
-        PyErr_Restore(ptype, pvalue, ptb);
+        // if we got a system exit exception (not a subclass of Exception),
```

Comment:
`//` comments are not used in psycopg codebase as per PEP 7

Reply:
sure